src/images/selector-brace.svg

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
When we run this through [ImageOptim](https://imageoptim.com/mac), we save a great 73% (of 3kb ;-DD). Unfortunately, I don't know of a Linux version of this tool other than the web service and the command line tools that is uses internally.

```<svg height="8" viewBox="0 0 950 8" width="950" xmlns="http://www.w3.org/2000/svg"><path d="m456.31757.26744101 2.98373.50200063 2.14147 1.16457056 5.72906 4.6376092 1.8659 1.1762717 1.68266-.2059553 7.10179-5.6165754 2.70389-1.21343614 2.09235-.49014214 3.2992-.31122199-31.15484.0885295z" fill="#fff"/><path d="m-1.1877132 8.0203719h953.3323832l.0528-7.7637167-469.57905-.0402362-2.09235.49168576-2.70389 1.21725764-5.0363 4.0573676-2.06549 1.5768963-1.68266.206604-1.8659-1.1799762-5.72906-4.6522146-2.14147-1.16823822-2.98373-.5035816-457.4997262-.005564" fill="#f5f5f5"/><path d="m-275.35003 535h466.79303c2.31239 0 4.64642.62379 6.44055 2.08264l5.83879 4.74765c1.00356.81602 2.50658.81247 3.51302 0l5.88109-4.74765c1.79927-1.4525 4.12816-2.08264 6.44055-2.08264h456.98426" fill="none" stroke="#979797" stroke-width=".5" transform="matrix(-1 0 0 1 675.13579 -534.743471)"/></svg>```

Reply:
Nice, good hint - didn't know it can also optimize SVGs! Internally it uses [svgo](https://www.npmjs.com/package/svgo), which I've used now :)